src/db_init.h

```diff
@@ -25,7 +25,7 @@
  * version of the database? If yes, then it is a minor upgrade, if no, then it
  * is a major upgrade. In other words minor version upgrades permit downgrading
  * forked-daapd after the database was upgraded. */
-#define SCHEMA_VERSION_MAJOR 19
+#define SCHEMA_VERSION_MAJOR 20
```

Comment:
The update is backwards compatible, i will change it to 19.01. 

Only the "INSERT INTO admin (key, value) VALUES ('queue_version', '0');" query leads to duplicate entries for the key "queue_version" if the update is run twice on the same db. The issue here is that the admin table does not define the key column as the primary key. An "insert or update" will also lead to duplicate entries because sqlite can not identify that the row with the same key value should be updated.